s3torchconnector/tst/e2e/conftest.py

```diff
@@ -46,6 +46,9 @@ def __init__(
     def s3_uri(self):
         return f"s3://{self.bucket}/{self.prefix}"
 
+    def is_standard_storage(self):
+        return self.storage_class != "EXPRESS_ONEZONE"
```

Comment:
I'd probably change this to `is_express_storage` and then the following usage to `if not checkpoint_directory.is_express_storage()`